django/core/management/commands/makemessages.py

```diff
@@ -627,11 +627,11 @@ def write_po_file(self, potfile, locale):
                 msgs = fp.read()
             if not self.invoked_for_django:
                 msgs = self.copy_plural_forms(msgs, locale)
-        msgs = normalize_eols(msgs)
-        msgs = msgs.replace(
-            "#. #-#-#-#-#  %s.pot (PACKAGE VERSION)  #-#-#-#-#\n" % self.domain, "")
-        with open(pofile, 'w', encoding='utf-8') as fp:
-            fp.write(msgs)
+            msgs = normalize_eols(msgs)
```

Comment:
Wouldn't we still want to `normalize_eols()`, even for existing files? 
(https://code.djangoproject.com/ticket/26645 seems to describe why this was introduced in the first place)

Reply:
Yes that seems right. I missed this as I couldn't find any tests for this.